### PR TITLE
Hugo documentation changes and copyright notice changed to 2016

### PR DESCRIPTION
**Pull request by hacdias**
I also ask you to review the docs to make sure it doesn't have any errors. When you've time, I would like to now if you can clean the cache to update caddy-hugo to the new version.

### CODE REVIEW

File: public/docs/hugo.html [hunk 1]
```diff
@@ -14,11 +14,13 @@
 			</div>
 
 			<p>
-				hugo allows you to deploy and manage a static site with an web-based admin interface where you can edit content, schedule posts, upload files, etc. The admin interface will be available at /admin on your website. It is built to be used with <a href="http://gohugo.io/">Hugo</a> - which is bundled with the plugin so you don't have to install it separately.
+				<strong>hugo fills the gap between Hugo and the browser.</strong> <a href="http://gohugo.io/">Hugo</a> is an easy, blazing fast and awesome static website generator. This plugin fills the gap between Hugo and the end-user, providing you an web interface to manage the whole website.
```
Comment (mholt) [inline public/docs/hugo.html]: Let's lose the `<strong>` tags here. Also, these are documentation pages, not marketing pages, so we can lose the promotional language, like "blazing fast and awesome" -- just try to stick to simply saying what it does.

File: public/docs/hugo.html [hunk 2]
```diff
@@ -48,6 +42,10 @@
 			<h3 id="examples">Examples</h3>
 
 			<p>
+				If you still <strong>don't</strong> have an Hugo website, don't worry. This plugin will auto-generate it for you. But that's not everything. I recommend you to take a look at Hugo <a href="http://gohugo.io/themes/overview/">documentation</a> to learn more about themes, content types, and so on.
```
Comment (mholt) [inline public/docs/hugo.html]: Again, we can lose the strong tags, and maybe add the word "already". "If you don't already have a Hugo website, ..."

**Comment by mholt** [comment]
Run the PNG image through tinypng.com.

Looking good!

**Comment by hacdias** [comment]
Done  ;)

**Comment by mholt** [comment]
Great -- and I'll need to update the plugin on the build server before this is released. Thanks!

**Comment by hacdias** [comment]
Thansk :)

**Pull request merged**